### ISSUE TITLE
"Map object 'cilium_ipv4_frag_datagrams' rejected: Unknown error 524 (524)!

### ISSUE THREAD

**Issue by tgraf** [cilium/cilium#10925]
Seeing this in recent master. It seems to recover after a restart.

```
ium/bpf/include -D__NR_CPUS__=2 -DENABLE_ARP_RESPONDER=1 -DHANDLE_NS=1 -DFROM_HOST -DSECLABEL=1 -DBPF_PKT_DIR=0 '-DNODE_MAC={.addr={0x8e,0xdc,0x4e,0x28,0xc5,0x82}}' -DCALLS_MAP=cilium_calls_netdev_ns_1 -c /var/lib/cilium/bpf/bpf_netdev.c -o -" subsys=datapath-loader
level=warning msg="+ tc qdisc replace dev cilium_host clsact" subsys=datapath-loader
level=warning msg="++ grep -v 'pref 1 bpf chain 0 $\\|pref 1 bpf chain 0 handle 0x1'" subsys=datapath-loader
level=warning msg="++ tc filter show dev cilium_host egress" subsys=datapath-loader
level=warning msg="+ '[' -z '' ']'" subsys=datapath-loader
level=warning msg="+ cilium-map-migrate -s bpf_host.o" subsys=datapath-loader
level=warning msg="+ set +e" subsys=datapath-loader
level=warning msg="+ tc filter replace dev cilium_host egress prio 1 handle 1 bpf da obj bpf_host.o sec from-netdev" subsys=datapath-loader
level=warning msg="Map object 'cilium_ipv4_frag_datagrams' rejected: Unknown error 524 (524)!" subsys=datapath-loader
level=warning msg=" - Type:         9" subsys=datapath-loader
level=warning msg=" - Identifier:   0" subsys=datapath-loader
level=warning msg=" - Pinning:      2" subsys=datapath-loader
level=warning msg=" - Size key:     12" subsys=datapath-loader
level=warning msg=" - Size value:   4" subsys=datapath-loader
level=warning msg=" - Max elems:    8192" subsys=datapath-loader
level=warning msg=" - Flags:        0x1" subsys=datapath-loader
level=warning subsys=datapath-loader
level=warning msg="Error loading maps into kernel!" subsys=datapath-loader
level=warning msg="Error fetching ELF ancillary data!" subsys=datapath-loader
level=warning msg="Unable to load program" subsys=datapath-loader
level=warning msg="+ RETCODE=1" subsys=datapath-loader
level=warning msg="+ set -e" subsys=datapath-loader
level=warning msg="+ cilium-map-migrate -e bpf_host.o -r 1" subsys=datapath-loader
level=warning msg="+ return 1" subsys=datapath-loader
level=error msg="Error while initializing daemon" error="exit status 1" subsys=daemon
level=fatal msg="Error while creating daemon" error="exit status 1" subsys=daemon
level=debug msg="Controller func execution time: 15.044µs" name=metricsmap-bpf-prom-sync subsys=controller uuid=2a6c9d38-7aaf-11ea-a138-0258a0fe1826
```

**Comment by qmonnet** [comment]
Do you know if the option `preallocate-bpf-maps` was set to `false`, by any chance?

**Comment by tgraf** [comment]
> Do you know if the option `preallocate-bpf-maps` was set to `false`, by any chance?

I've been using the default installation setup, so, yes:
```
  preallocate-bpf-maps: "false"
```

**Comment by qmonnet** [comment]
Thanks, so in that case this should be fixed with #10927 and in particular [commit a61dffc571b4](https://github.com/cilium/cilium/commit/a61dffc571b4336c655cc659b0cbb10d86c88d83). I'm closing the issue.
